### ISSUE TITLE
May I ask why the loss of my training set is normal, but the dice loss of the validation set is 0, and the labeled loss is negative?

### ISSUE THREAD

**Issue by sugarhaterwow** [alimottaghi/brain-segmentation#1]
- Train metrics: dice score: 0.198 ; labeled loss: 0.959 ; unlabeled loss: 0.000 ; loss: 0.959
- Eval metrics : dice score: 0.000 ; labeled loss: -0.451
